stingray/pulse/tests/test_modeling.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from stingray.pulse.modeling import fit_sinc, fit_gaussian
+
+np.random.seed(0)
+
```

Comment:
What do you mean? fit_sinc and fit_gaussian are tested in each test :) 
Maybe you mean the model functions?